osutil/disks/labels.go

```diff
@@ -0,0 +1,48 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package disks
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"unicode/utf8"
+)
+
+// EncodeHexBlkIDFormat encodes a name for use as a partition or filesystem
+// label symlink by udev. The result matches the output of blkid_encode_string()
+// from libblkid.
+func EncodeHexBlkIDFormat(in string) string {
```

Comment:
BlkIDEncodeLabel ?  it would be closer to the original function name and the doc comment

Reply:
sure I'll do this in a followup since we're almost all green here